src/main/scala/tech/cryptonomic/conseil/routes/openapi/DataJsonSchemas.scala

```diff
@@ -9,6 +9,20 @@ import tech.cryptonomic.conseil.tezos.Tables.{AccountsRow, BlocksRow, OperationG
 /** Trait containing Data endpoints JSON schemas */
 trait DataJsonSchemas extends generic.JsonSchemas {
 
+  /** API field schema */
+  implicit val fieldSchema: JsonSchema[Field] //=
+//    genericJsonSchema[Field]
```

Comment:
Is there anything that needs addressing here?

Reply:
forgot to remove unused code/comments